mongoalchemy/ops.py

```diff
@@ -100,7 +101,18 @@ def __init__(self, trans_id, session, document, safe):
     def execute(self):
         self.ensure_indexes()
         kwargs = safe_args(self.safe)
-        return self.collection.save(self.data, **kwargs)
+        if self.shard_key:
```

Comment:
Should this be protected by an option? Given that we currently don't use a sharded database it seems unfortunate to crash on all queries that don't yet support it